requirements.txt

```diff
@@ -1,27 +1,26 @@
 boto3==1.7.37
 click==6.7
 configparser==3.7.4
-cython==0.26
+cython
 future==0.16.0
-INDI-Tools==0.0.6
+git+git://github.com/shnizzedy/INDI-Tools.git@3a7b80558bd866bf57e2d911f75aeb8bd62b2723
```

Comment:
This version is different from that used in `info.py`. Any reason for the discrepancy?

Reply:
In this PR `info.py` does not specify a version: https://github.com/FCP-INDI/C-PAC/blob/e9b866eb22e9cd43b5d4959e6a8f20b0ffb2b9bb/CPAC/info.py#L140

INDI-Tools was [migrated to Python 3 in 2018](https://github.com/FCP-INDI/INDI-Tools/commit/f052440335913d03f89b75504bc9a6577877308b) but [the PyPI distribution](https://pypi.org/project/INDI-Tools/) hasn't been updated yet. I made [a fork](https://github.com/FCP-INDI/INDI-Tools/pull/1/files) to [increment the version and classifiers](https://github.com/FCP-INDI/INDI-Tools/pull/1/files) and [install from in the meantime](https://github.com/FCP-INDI/C-PAC/pull/1200/commits/e368ba780c8a4e02f4d81bf0810268f2c522f784) until we update the PyPI distro.

Reply:
That makes sense to maintain the specific version used here until the package is updated in PyPI.